java/src/jmri/jmrix/can/cbus/CbusMessage.java

```diff
@@ -481,7 +481,12 @@ static public CanMessage getBootNop(int a, int header) {
         m.setElement(1, (a / 256) & 0xFF);
         m.setElement(2, (a / 65536) & 0xFF);
         m.setElement(3, 0);
-        m.setElement(4, 0x0D);
+        //m.setElement(4, 0x0D);
+        m.setElement(4, CbusConstants.CBUS_BOOT_MODE_ACK
```

Comment:
The reason CI fails is that the ACK bit is not set. So by including the ACK bit, you have changed what the code does.

Reply:
I am happy that this change has no bad side effects. The tests can be changed to reflect the new behaviour